client/src/arrowAfter.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import $ from 'jquery';
+import styled from 'styled-components';
+
+const Div = styled.div`
+  border:1px solid black;
+  width:14%;
+  height:100%;
+`;
+
+
+class ArrowAfter extends React.Component {
+  
+  constructor(props) {
```

Comment:
If you're not using the constructor, you need to change this to a stateless component